File: android/BertSpeak/app/src/main/java/chuckcoughlin/bert/waveform/RendererFactory.java
/**
 * @See: Mark Allison @ https://github.com/StylingAndroid/Visualizer
 */
package chuckcoughlin.bert.waveform;
import android.support.annotation.ColorInt;

import chuckcoughlin.bert.waveform.SimpleWaveformRenderer;
import chuckcoughlin.bert.waveform.WaveformRenderer;

public class RendererFactory {
    public WaveformRenderer createSimpleWaveformRenderer(@ColorInt int foreground, @ColorInt int background) {
        return SimpleWaveformRenderer.newInstance(background, foreground);
    }
}

File: BertSpeak/app/src/main/kotlin/chuckcoughlin/bertspeak/waveform/RendererFactory.kt
/**
 * @See: Mark Allison @ https://github.com/StylingAndroid/Visualizer
 */
package chuckcoughlin.bertspeak.waveform

import androidx.annotation.ColorInt

class RendererFactory {
    fun createSimpleWaveformRenderer(
        @ColorInt foreground: Int,
        @ColorInt background: Int
    ): WaveformRenderer {
        return SimpleWaveformRenderer.Companion.newInstance(background, foreground)
    }
}
